cross/cups/Makefile

```diff
@@ -28,6 +28,9 @@ ADDITIONAL_LDFLAGS = -Wl,-rpath,$(INSTALL_DIR)/var/packages/$(PKG_NAME)/target/l
 
 PATCHES_LEVEL = 1
 
+# disable parallel build. ppdc sub package will fail building in parallel on some platforms
+PARALLEL_MAKE = 0
+
```

Comment:
The regular way to disable parallel builds in gnu make is to define the notparallel target like this:
```
.NOTPARALLEL:
```

Reply:
well PARALLEL_MAKE = 0 didnt work either.

how does 

.NOTPARALLEL: 

works?

Reply:
for cross/cups

do i make my own PRE_COMPILE_TARGET

and declare .NOTPARALLEL: in it?

Reply:
just put .NOTPARALLEL:  in cross/cups/Makefile on a separate line

the same is used in `/spksrc/mk/spksrc.native-cc.mk` and other *.mk files.

Reply:
Hmm still fails

Compiling ppdpo.cxx...
Linking ppdpo...
Compiling genstrings.cxx...
Linking genstrings...
Generating localization strings...
./genstrings: error while loading shared libraries: libgnutls.so.30: cannot open shared object file: No such file or directory
make[4]: *** [Makefile:189: genstrings] Error 127
make[4]: Leaving directory '/spksrc/spk/java-11-openjdk/work-evansport-7.0/cups-2.3.3/ppdc'
make[3]: *** [Makefile:42: all] Error 1
make[3]: Leaving directory '/spksrc/spk/java-11-openjdk/work-evansport-7.0/cups-2.3.3'

Reply:
maybe a ldconfig is missing?